org.jacoco.core/src/org/jacoco/core/internal/analysis/filter/KotlinGeneratedFilter.java

```diff
@@ -23,6 +23,10 @@
 
 	static final String KOTLIN_METADATA_DESC = "Lkotlin/Metadata;";
 
+	static boolean isKotlinClass(final IFilterContext context) {
```

Comment:
As this method is used by other filters too, I wonder whether the `GeneratedFilter` is the right place. Maybe a separate `KotlinFilterUtils` class would be a better place?